### ISSUE TITLE
Image Button inconsistency from empty naming attribute

### ISSUE THREAD

**Issue by WilcoFiers** [auto-wcag/auto-wcag#2293]
## Accessibility support

The [input type="image" Accessible Name Computation algorithm](https://www.w3.org/TR/html-aam/#input-type-image-accessible-name-computation) uses the first defined attribute, even if the attribute has an empty value. In contrast, some user agents and assistive technologies ignore empty attributes.

## ACT Rules

- [Image button has non-empty accessible name - 59796f](https://www.w3.org/WAI/standards-guidelines/act/rules/59796f/proposed/#accessibility-support)

### Test case 1

Testing image button accessible name computation across different naming methods

```html
TODO
```

#### Test instructions

TODO

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case 1

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Screen reader: ..., version ...
- Test result: Passed / Failed (if failed, how was it different from expected?)
```

**Comment by shunguoy** [comment]
1. based on the [acc name calculation spec](https://www.w3.org/TR/accname-1.2/), an aria-label or aria-labbeledby attribute will be ignored in the acc name calculation if its value is undefined or empty string when trimmed of whitespace.
2. The last resort ([step 5](https://www.w3.org/TR/html-aam/#input-type-image-accessible-name-computation)) of an acc name for an input image button is from an [implementation defined](https://infra.spec.whatwg.org/#implementation-defined) string, which is "Submit" for Chrome regardless of the existence of the image src, and "Submit Query" for FF only if the src image doesn't exist.
   The descriptions for each of the three Failed examples state that each button has an accessible name “Submit Query” (from the default internal implementation), however, they still fail without explanations of why.
